Sources/AsyncServer.swift

```diff
@@ -1,9 +1,9 @@
 public protocol AsyncServer {
-    func serve(responder: AsyncResponder, on host: String, at port: Int) throws
+    func serve(_ responder: AsyncResponder, on host: String, at port: Int) throws
```

Comment:
Since responder is often router, I think `serve(responder: router … ` will fit here nicely, no tautology.